services/backend/src/routes/connectivity.js

```diff
@@ -11,7 +11,7 @@ module.exports = (errorHandler, connectivityService) => {
    * @apiDescription
    * Area distribution for each category of protected area connectivity in a given area
    *
-   * Value calculated for 2018
+   * Value calculated for 2020
    *
    * @apiParam (Query params) {String|Number} areaType area type
```

Comment:
According previous reviews areaType is String and the one that could be both String or Number is areaId

Reply:
Done